consensus/obcpbft/broadcast.go

```diff
@@ -67,35 +73,71 @@ func (b *broadcaster) Wait() {
 	b.closed.Wait()
 }
 
-func (b *broadcaster) unicastOne(msg *pb.Message, dest uint64, wait chan bool) {
+func (b *broadcaster) drainerSend(dest uint64, send *sendRequest, printedValidatorNotFound bool) bool {
```

Comment:
Agreed, I considered flipping this, and will do so now.  The status is only used to avoid flooding our logs when some peer is not connected.